recipe/meta.yaml

```diff
@@ -28,58 +28,60 @@ app:
 
 requirements:
   host:
-    - jupyter-packaging 0.12.0
-    - packaging
+    - jupyter-packaging
```

Comment:
Is jupyter-packaging no more required https://github.com/jupyterlab/jupyterlab/blob/2c9dff7e8343400b5d7d65839cd2f48e8d5ddb69/pyproject.toml#L5

Reply:
I think you are right, I'll test and remove it if not needed. Thanks.